docker-compose-gpu-runner-cuda-12-0.yml

```diff
@@ -1,14 +1,27 @@
 version: '3.2'
 services:
-  geth:
-    image: "matterlabs/geth:latest"
-    environment:
-      - PLUGIN_CONFIG
+  reth:
+    restart: always
+    image: "ghcr.io/paradigmxyz/reth:v0.2.0-beta.1"
+    volumes:
+      - type: bind
+        source: ./volumes/reth/data
+        target: /rethdata
+      - type: bind
+        source: ./volumes/reth/logs
+        target: /rethlogs
+      - type: bind
+        source: ./etc/test_config/
```

Comment:
IIRC `test_config` was introduced as a folder for some test configurations. Keeping reth configuration here is counterintuitive.

I suggest doing the following:
1. Create folder `/etc/reth`
2. Create `/etc/reth/README.md` with a couple of sentences on what this folder represents (e.g. data to run reth locally, including folders and files mounted to the reth container as read-only volumes). Please check it with grammarly and check that it doesn't look clunky😅 
3. Create `/etc/reth/chaindata` with the contents that are intended to be mounted to reth.
4. Here and in other docker-compose files use `/etc/reth/chaindata`.